easybuild/framework/easyconfig/parser.py

```diff
@@ -45,6 +45,59 @@
 # alternate easyconfig parameters, and their non-deprecated equivalents
 ALTERNATE_PARAMETERS = {
     # <new_param>: <equivalent_param>,
+    'build_deps': 'builddependencies',
+    'build_in_install_dir': 'buildininstalldir',
+    'build_opts': 'buildopts',
+    'build_stats': 'buildstats',
+    'clean_up_old_build': 'cleanupoldbuild',
+    'clean_up_old_install': 'cleanupoldinstall',
+    'configure_opts': 'configopts',
+    'deps': 'dependencies',
+    'doc_paths': 'docpaths',
+    'doc_urls': 'docurls',
+    'dont_create_install_dir': 'dontcreateinstalldir',
+    'exts_class_map': 'exts_classmap',
+    'exts_default_class': 'exts_default_class',
```

Comment:
should be `exts_defaultclass` on right-hand side:
```suggestion
    'exts_default_class': 'exts_defaultclass',